Tests/AK/TestComplex.cpp

```diff
@@ -9,9 +9,47 @@
 #include <AK/Complex.h>
 
 static_assert([]() {
+    auto const z = Complex<double>();
     auto const r = complex_real_unit<double>;
     auto const i = complex_imag_unit<double>;
 
+    auto basic_math = [&](auto const& c) {
+        auto n = c;
+        assert(c == c);
+        assert(!(c != c));
+
+        assert(c + c == 2 * c);
+        assert(c + 0 == c);
+        assert(0 + c == +c);
+        assert((n += 0) == c);
+
+        assert(c - c == 0);
+        assert(c - 0 == c);
+        assert(0 - c == -c);
+        assert((n -= 0) == c);
+
+        assert(c * r == c);
+        assert(c * 1 == c);
+        assert(1 * c == c);
+        assert((n *= 1) == c);
+
+        assert(c / r == c);
+        assert(c / 1 == c);
+        assert((n /= 1) == c);
+
+        assert(+c == 0 + c);
+        assert(-c == 0 - c);
+    };
+
+    basic_math(z);
+    //assert(1 / z == inf);
```

Comment:
Pls no commented code
If this is case that's failing but is expected to pass, please add a fixme.